### PR TITLE
Fix Envoy Setup Script bug

### PR DESCRIPTION
**Pull request by koolzz**
Signed-off-by: Nick Yurchenko <koolzz@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/deploy/roles/magma/files/configure_envoy_namespace.sh [hunk 1]
```diff
@@ -15,48 +15,48 @@ envoy_ns="envoy_ns1"
...
+  /sbin/ip netns exec  $envoy_ns ifconfig "$envoy_dp_dev"_ns    "$envoy_dp_dev_ip"/24 up
 
-  ip netns exec  $envoy_ns ifconfig lo up
+  /sbin/ip netns exec  $envoy_ns ifconfig lo up
 
-  ip netns exec  $envoy_ns ip route add default via $Router_IP
+  /sbin/ip netns exec  $envoy_ns ip route add default via $Router_IP
 
-  ip netns exec  $envoy_ns ip neigh replace $Router_IP  lladdr $Router_mac dev "$envoy_dp_dev"_ns
+  /sbin/ip netns exec  $envoy_ns ip neigh replace $Router_IP  lladdr $Router_mac dev "$envoy_dp_dev"_ns
```
Comment (pshelar) [inline lte/gateway/deploy/roles/magma/files/configure_envoy_namespace.sh]: There is another `ip` command nested, can you replace that too.

File: lte/gateway/deploy/roles/magma/files/configure_envoy_namespace.sh [hunk 2]
```diff
@@ -15,48 +15,48 @@ envoy_ns="envoy_ns1"
...
 
-  ip netns exec  $envoy_ns iptables -t mangle -I PREROUTING -p tcp --dport 80 -j MARK --set-mark 1
-  ip netns exec  $envoy_ns iptables -t mangle -I PREROUTING -p tcp --sport 80 -j MARK --set-mark 1
+  /sbin/ip netns exec  $envoy_ns iptables -t mangle -I PREROUTING -p tcp --dport 80 -j MARK --set-mark 1
+  /sbin/ip netns exec  $envoy_ns iptables -t mangle -I PREROUTING -p tcp --sport 80 -j MARK --set-mark 1
```
Comment (pshelar) [inline lte/gateway/deploy/roles/magma/files/configure_envoy_namespace.sh]: same here for iptables.

File: lte/gateway/deploy/roles/magma/files/configure_envoy_namespace.sh [hunk 3]
```diff
@@ -15,48 +15,48 @@ envoy_ns="envoy_ns1"
...
+  /sbin/ip link set dev "$envoylink_cntr"_ns  netns $envoy_ns
 
-  ip netns exec  $envoy_ns ip link set dev "$envoy_dp_dev"_ns address $mac_addr
+  /sbin/ip netns exec  $envoy_ns ip link set dev "$envoy_dp_dev"_ns address $mac_addr
   # namespace configi
-  ip netns exec  $envoy_ns ifconfig "$envoylink_cntr"_ns  "$envoy_ip_cntr"/24 up
+  /sbin/ip netns exec  $envoy_ns ifconfig "$envoylink_cntr"_ns  "$envoy_ip_cntr"/24 up
 
-  ip netns exec  $envoy_ns ifconfig "$envoy_dp_dev"_ns    "$envoy_dp_dev_ip"/24 up
+  /sbin/ip netns exec  $envoy_ns ifconfig "$envoy_dp_dev"_ns    "$envoy_dp_dev_ip"/24 up
```
Comment (pshelar) [inline lte/gateway/deploy/roles/magma/files/configure_envoy_namespace.sh]: same abs path required for nested ifconfig.

**Review by pshelar — CHANGES_REQUESTED** [review]

**Review by pshelar — APPROVED** [review]
LGTM

**Review by ardzoht — APPROVED** [review]

**Pull request merged**